src/main/java/cat/udl/eps/softarch/unicloud/domain/User.java

```diff
@@ -34,11 +34,37 @@
 	@Column(unique = true)
 	private String email;
 
+	public String getUsername() {
```

Comment:
There's no need to implement getters and setters, they are automatically provided by using Lombok's annotation @Data: https://projectlombok.org/features/Data

You can remove these lines implementing setters and getters.